dapr/aio/clients/grpc/client.py

```diff
@@ -1103,6 +1110,278 @@ async def unlock(
         return UnlockResponse(status=UnlockResponseStatus(response.status),
                               headers=await call.initial_metadata())
 
+    async def start_workflow(
+            self,
+            workflow_component: str,
+            workflow_name: str,
+            input: Union[Any, bytes, None] = None,
+            instance_id: Optional[str] = None,
+            workflow_options: Optional[Dict[str, str]] = dict()) -> StartWorkflowResponse:
+        """Starts a workflow.
+
+            Args:
+                workflow_component (str): the name of the workflow component
+                                    that will run the workflow. e.g. `dapr`.
+                workflow_name (str): the name of the workflow that will be executed.
+                input (Union[Any, bytes, None]): the input that the workflow will receive.
+                instance_id (Optional[str]): the name of the workflow instance,
+                                    e.g. `order_processing_workflow-103784`.
+                workflow_options (Optional[Dict[str, str]]): the key-value options
+                                    that the workflow will receive.
+
+            Returns:
+                :class:`StartWorkflowResponse`: Instance ID associated with the started workflow
+        """
+        # Warnings and input validation
+        warn('The Workflow API is an Alpha version and is subject to change.',
+             UserWarning, stacklevel=2)
+        validateNotBlankString(instance_id=instance_id,
+                               workflow_component=workflow_component,
+                               workflow_name=workflow_name)
+
+        if instance_id is None:
+            instance_id = str(uuid.uuid4())
+
+        if isinstance(input, bytes):
+            encoded_data = input
+        else:
+            encoded_data = json.dumps(input).encode("utf-8") if input is not None else bytes([])
```

Comment:
This will cause completely arbitrary results converting who knows what kind of object to bytes this way.

I strongly recommend removing this together with the `Any` type and letting a user handle the bytes conversion themselves (so that they know what conversion they used)